### ISSUE TITLE
Configure the Localization server module

### ISSUE THREAD

**Issue by olafgrabienski** [backdrop-ops/localization#1]
To start using https://localize.backdropcms.org as a localization server, we have to configure the *Localization server* module at `admin/l10n_server`.

**Relevant Gitter discussion:**

> I could (...) try to configure the translation server. For the moment, I've found only some docs that seems outdated (http://hojtsy.hu/blog/2010-may-05/using-drupal-collaborative-software-translation-tool), but I'll search for more. (@olafgrabienski)

> file an issue for configuring the l10n_server (...) and you and I can stumble through it (@serundeputy)

> I never configured a l10n server before either, but I try to help as much as I can too. (@klonos)

So, let's start!

**Comment by olafgrabienski** [comment]
Here some docs:

- Project page: https://www.drupal.org/project/l10n_server
- Blog article (Drupal 6): http://hojtsy.hu/blog/2010-may-05/using-drupal-collaborative-software-translation-tool
- Readme file (Drupal 7): http://cgit.drupalcode.org/l10n_server/tree/README.txt?id=refs/heads;id2=7.x-1.x
- Localize.drupal.org administrator guide: https://www.drupal.org/node/1439588

**Comment by olafgrabienski** [comment]
Yay, I've made some steps to configure the Localization server module, added a project (Drupal core) and three languages as examples, and imported the relevant language files from drupal.org.

At the moment, anonymous users can explore translations. Community roles and permissions are to be done.

URL: https://localize.backdropcms.org/translate

Configuration steps so far:

- `admin/l10n_server/connectors`  
*Gettext files from file uploads*
- `admin/l10n_server/projects/add`  
Added the project *Drupal* (for Drupal core).
- `admin/l10n_server/projects/releases/drupal/add` 
Added release: *7.54*.  
To generate a (English) *Source file* for translations, I took a German `.po` and replaced the translations with empty translations strings.
- `admin/people/create` 
Created an authenticated Drupal user named "Multiple contributors" to be able to attribute bulk translations (file imports) to a *not personal* username.
- `admin/l10n_server/l10n_community`  
*Account associated with bulk imports: "Multiple contributors"*  
Unfortunately, I discovered the last two steps only after the first language import, resulting in the German language translations attributed to my personal username.
- `admin/config/regional/language` 
Added German, Greek and Spanish as example languages.
- `translate/languages/[de|el|es]/import`  
Imported `.po`files for the example languages.
- `admin/l10n_server/packager`  
"Repackaged" once. As a result, you can download translations.
- `admin/people/permissions`  
Permissions for Anonymous user:  
*access localization community*, *browse translations*

@todo: Add a role and permissons (e.g. *submit suggestions*) for translation community members. Update: that will be another issue, see Roadmap, issue #5 .

**Comment by olafgrabienski** [comment]
Interestingly, the three example languages seem to be translated completely (which won't be the case with Backdrop later). Maybe I should add a less translated language for demonstration purposes of the community translation interface ...

**Comment by olafgrabienski** [comment]
Imported Swahili which has only 8 (!) translations.

@todo to have a download of the Swahili `.po` file: "Repackage now" on `admin/l10n_server/packager`

**Comment by olafgrabienski** [comment]
I've configured the translation server now to handle Backdrop language files as well!

- `admin/l10n_server/projects/add`  
Added the project *Backdrop CMS*
- `admin/l10n_server/projects/releases/backdropcms/add` 
Added release: *1.6.1*.  
To generate the English *Source file* for translations, I exported a `.pot` (language template) file of a Backdrop test site. (See also issue #6, *Generate English Backdrop language files regularly*.)
- admin/l10n_server/packager  
*Backdrop CMS > Repackage now* (provides translation downloads)

Looking at https://localize.backdropcms.org/translate/projects/backdropcms I have the impression that Drupal 7.54 strings which exist in Backdrop have been automatically adopted by Backdrop CMS:

![screen-localize-backdrop-translations](https://cloud.githubusercontent.com/assets/14188307/23385683/ce4ddc48-fd51-11e6-9654-f79a96f8f92d.png)

**Comment by quicksketch** [comment]
Wow this is great @olafgrabienski! Fantastic work so far. :smile:

**Comment by olafgrabienski** [comment]
Closing this issue; the basic configuration is done for quite a while.
